README.md

```diff
@@ -230,6 +230,7 @@
 - 🧩 [Windstrap](https://windstrap.netlify.app) - Tailwind CSS with Bootstrap JS.
 - 🧩 [WickedBlocks](https://blocks.wickedtemplates.com/) - Collection of more than 120 layout blocks and components built with Tailwind CSS.
 - 🧩 [Daisy UI](https://github.com/saadeghi/daisyui) - UI Components for Tailwind CSS.  
+- 🧩 [Tail-kit](https://tailwind-kit.com/) - 200+ free and open-source components.
```

Comment:
After talking with Adam, the domain still violates brand usage guidelines. Once that is resolved and we don't see any violations of the [Tailwind Brand Usage Guidelines](https://tailwindcss.com/brand#usage), we'll happily accept the PR.